### DISCUSSION TITLE
Hello,

### DISCUSSION THREAD

**Discussion by OZaher** [Open-Systems-Pharmacology/Forum#1056] [Questions & Problems] [answered]
I am working on a project to simulate a dosage form for an oral drug.
I have entered the physiochemical properties for the drug and when running simulations I got no elimination phase for the drug, I mean the drug plasma concentration remains the same after small decrease after the peak.
I am thinking to look at the metabolizing enzymes , if anyone have a better idea will be grateful.

**Comment by tobiasK2001** [comment, marked as answer]
Hi,

yes adding metabolism via metabolic enzymes to see Elimination is always a good idea. You might keep in mind that also non-metabolic processes like renal or biliary clearance  can contribute to elimination. Depending on the granularity of your knowledge they can be implemented in different ways. Check out this video as an example for ciprofloxacin using different path of elimination. https://github.com/Open-Systems-Pharmacology/ISSX-2017-Cologne/tree/master/Session%202.%20Building%20a%20PBPK%20model

Best, Tobias

**Reply by OZaher** [reply]
Thanks
